bundle/bundle.source.js

```diff
@@ -9,6 +9,7 @@
     units: require('../lib/units')(),
     utils: require('../lib/utils')(),
     profile: require('../lib/profilefunctions')(),
+    language: require('../lib/language')(serverSettings.defaults.language),
```

Comment:
don't think we should expect `serverSettings` to be available globally here, maybe instead do `language: require('../lib/language')` and let the client initialize it